apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -85,16 +85,17 @@ export default class CoreLibrary {
       return;
     }
 
-    this.speechBubbles.forEach(({text, sprite}) => {
+    this.speechBubbles.forEach(({text, sprite, style}) => {
```

Comment:
Similar to my comment on padding - `style` in our codebase normally refers to the CSS objects - can we use a different name here to differentiate that this is "say/think"?

Reply:
All feedback addressed, PTAL! @epeach